app/src/main/res/values-it/strings.xml

```diff
@@ -76,66 +76,66 @@
-    <string name="background_color">Background Color</string>
-    <string name="pick_color">Pick Color</string>
-    <string name="input_color">Input Color</string>
-    <string name="command_color">Command Color</string>
-    <string name="normal_text_and_arrow_color">Normal Text and Arrow Color</string>
-    <string name="error_text_color">Error Text Color</string>
-    <string name="positive_text_color">Positive Text Color</string>
-    <string name="warning_text_color">Warning Text Color</string>
-    <string name="suggestions_color">Suggestions Color</string>
-    <string name="internet_speed_test">Internet Speed Test</string>
-    <string name="loading_please_wait">Loading...Please Wait.</string>
-    <string name="cmd_ai_title">ai [message]</string>
-    <string name="cmd_ai_help">A simple tool to access chatGPT from the terminal. Based on OpenAI\'s gpt-3.5-turbo, you can chat with your own AI assistant. An API key is required to be entered in \'settings\'. This command does not remember context, so the model will be unaware of your previous conversation.</string>
-    <string name="ai_msg_missing">Please specify the message to send to AI.</string>
-    <string name="no_ai_api_key_found">You probably forgot to provide an API key. Please enter it in \'settings\'.</string>
-    <string name="communicating_with_ai">Communicating with AI...</string>
-    <string name="no_reply_in_response">No reply found in the response</string>
-    <string name="no_reply_from_server">The server did not send a chat reply! Try again.</string>
-    <string name="ai_timeout">I lost my patience! Try again or try a request with a shorter expected output.</string>
-    <string name="ai_auth_failed">Authentication Failed. Make sure you used the correct API key in \'settings\'</string>
-    <string name="ai_error">An error occurred: %1$d</string>
+    <string name="don_t_ask_again">Non chiedere di nuovo</string>
+    <string name="done_will_never_ask_again">Fatto (￣┰￣*) Non chiederò più!!</string>
+    <string name="join_the_community">Unisciti alla community!</string>
+    <string name="community_description">Unisciti alla community per ottenere gli ultimi aggiornamenti su Yantra Launcher, fare domande, ottenere aiuto, discutere di nuove funzionalità e altro ancora!\n\nTutti là fuori sono appassionati di CLI😎 come te, quindi unisciti alla community e divertiti!</string>
+    <string name="take_me_there">Portami lì</string>
+    <string name="no_thanks">No grazie</string>
+    <string name="we_d_miss_you">Ci mancherai!\n༼☯﹏☯༽</string>
+    <string name="crash_title">Si è verificato un errore inaspettato!</string>
+    <string name="crash_message">Spiacente, Yantra Launcher sembra essere crashato! Invia un report allo sviluppatore per aiutare a debuggare il crash. Premendo Ok verrai indirizzato al tuo client email.</string>
+    <string name="crash_comment_prompt">Puoi aggiungere un commento opzionale qui per descrivere il crash:</string>
```

Comment:
Can you add a "prefer english if possible" kind of message in a good way to this?
For obvious reasons...

Reply:
Added "(preferibilmente in Inglese)" - "(preferably in English)"